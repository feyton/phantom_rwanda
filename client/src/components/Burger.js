import React from 'react';

const Burger = ({ styles, bgStyles }) => {
  return (
    <div className={bgStyles}>
      <div className={`${styles} h-1 w-7 bg-primary mt-1`} />
      <div className={`${styles} h-1 w-7 bg-primary mt-1`} />
      <div className={`${styles} h-1 w-7 bg-primary mt-1`} />
    </div>
  );
};

export default Burger;
