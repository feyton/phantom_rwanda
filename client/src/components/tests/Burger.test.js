import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';
import { MemoryRouter } from 'react-router-dom';
import Burger from '../Burger.js';

describe('Burger Component', () => {
  it('should have a burger component', () => {
    const wrapper = mount(
      <MemoryRouter>
        <Burger />
      </MemoryRouter>
    );
    const value = wrapper.children().length;
    expect(value).toEqual(1);
  });
  it('it should render the Burger component', () => {
    const burger = renderer
      .create(
        <MemoryRouter>
          <Burger />
        </MemoryRouter>
      )
      .toJSON();
    expect(burger).toMatchSnapshot();
  });
});
