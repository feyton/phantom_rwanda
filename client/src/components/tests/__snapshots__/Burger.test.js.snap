// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Burger Component it should render the Burger component 1`] = `
<div>
  <div
    className="undefined h-1 w-7 bg-primary mt-1"
  />
  <div
    className="undefined h-1 w-7 bg-primary mt-1"
  />
  <div
    className="undefined h-1 w-7 bg-primary mt-1"
  />
</div>
`;
